internal/output/vertical.go

```diff
@@ -11,30 +11,91 @@ import (
 )
 
 func PrintVerticalResults(vulnResult *models.VulnerabilityResults, outputWriter io.Writer) {
-	for i, result := range vulnResult.Results {
-		printVerticalHeader(result, outputWriter)
-		printVerticalVulnerabilities(result, outputWriter)
+	// Add a newline to separate results from logs.
+	fmt.Fprintln(outputWriter)
+	outputResult := BuildResults(vulnResult)
+	printSummary(outputResult, outputWriter)
+	if outputResult.IsContainerScanning {
+		printBaseImages(outputResult.ImageInfo, outputWriter)
+	}
 
-		if len(vulnResult.ExperimentalAnalysisConfig.Licenses.Allowlist) > 0 {
-			printVerticalLicenseViolations(result, outputWriter)
+	for i, ecosystem := range outputResult.Ecosystems {
+		fmt.Fprintf(outputWriter, "%s", text.FgGreen.Sprintf("%s\n\n", ecosystem.Name))
+		for j, source := range ecosystem.Sources {
+			printVerticalHeader(source, outputWriter)
+			printVerticalVulnerabilities(source, outputResult.IsContainerScanning, outputWriter)
+			if j < len(ecosystem.Sources)-1 {
+				fmt.Fprintln(outputWriter)
+			}
 		}
 
-		if i < len(vulnResult.Results)-1 {
+		if i < len(outputResult.Ecosystems)-1 {
 			fmt.Fprintln(outputWriter)
 		}
 	}
+
+	fmt.Fprintln(outputWriter)
+}
+
+func printSummary(result Result, out io.Writer) {
+	summary := fmt.Sprintf(
```

Comment:
This needs to be adjusted to properly handle pluralization in the vent of 1 thing - hopefully you can do that with just `output.Form`, but note "have fixes" is tricky:
  - if there's only one fixable vulnerability and one fix, it should "1 vulnerability has a fix available"
  - if there's only one fixable vulnerability, with multiple fixes, it should be "1 vulnerability has fixes available"
  - if there's multiple fixable vulnerabilities, regardless of the number of fixes, it should be "x vulnerabilities have fixes available"

(alternatively, you could instead say "can be fixed" which I believe is valid for all three situations regardless of number of fixes or vulnerabilities)

Reply:
Thanks for the suggestion. I didn't really handle any pluralization, and will add this to other output formats. Updated it using outptu.Form and decided to go with the simplest way that using "can be fixed"